contrib/doca_urom_ucc_plugin/DOCA_UROM_README.md

```diff
@@ -0,0 +1,145 @@
+# Plugin Description
```

Comment:
Plugin Description -> "How to enable and run collectives on NVIDIA DPUs?"

Reply:
done